src/test-gui/java/de/qabel/desktop/ui/LayoutGuiTest.java

```diff
@@ -32,6 +35,15 @@ private LayoutController createController() {
     }
 
     @Test
+    public void showAboutViewOnStart() {
+        clientConfiguration.onSelectIdentity(this::aboutViewIsShown);
+    }
+
+    private boolean aboutViewIsShown(Identity identity) {
+        return waitForNode(ABOUT_VIEW) != null;
```

Comment:
`waitForNode` in the AbstractGuiTest is deprecated. Move it to a LayoutPage and call page.waitForAboutView() or use an AboutPage that waits for that node on initialization (because it can only act valid if it's root node exists).
```JAVA
new AboutPage(...).waitFor();
```

Additionally, all these waitFor* methods should fail if they don't find what they are looking for. So there shouldn't be the necessity to compare them to null ... of they return null then it's a bug ... they should return that node (or void!?) or throw an exception.